pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/IndexLoadingConfig.java

```diff
@@ -105,7 +106,8 @@ public class IndexLoadingConfig {
    */
   public IndexLoadingConfig(InstanceDataManagerConfig instanceDataManagerConfig, TableConfig tableConfig,
       @Nullable Schema schema) {
-    extractFromInstanceConfig(instanceDataManagerConfig);
+    _instanceDataManagerConfig = instanceDataManagerConfig;
+    extractFromInstanceConfig();
     updateTableConfigAndSchema(tableConfig, schema);
```

Comment:
Let's change this back to `extractFromTableConfigAndSchema` and make it private to prevent people from using it